crates/uv/src/commands/venv.rs

```diff
@@ -162,9 +162,7 @@ async fn venv_impl(
         // Resolve the seed packages.
         let resolution = build_dispatch
             .resolve(&[
-                Requirement::from_str("wheel").unwrap(),
                 Requirement::from_str("pip").unwrap(),
-                Requirement::from_str("setuptools").unwrap(),
```

Comment:
This list should be conditional on the Python version being used

Reply:
Thank you for reviewing this
I've updated it with a conditional check